config/katello.migrations/171030135054-foreman-proxy-update.rb

```diff
@@ -0,0 +1,5 @@
+# foreman_proxy defaults for templates
+
+answers['foreman_proxy']['http'] = true if answers['foreman_proxy'].key?('http')
+answers['foreman_proxy']['http_port'] = '8000' if answers['foreman_proxy'].key?('http_port')
```

Comment:
I think this needs to be an unless. I believe the test is failing, because at this point, the answers file has not been "fully" expanded, thus when it evaluates whether or not answers['foreman_proxy'] has 'http_port' as a key, it fails because it's not in the pre-changed answer file.